Resources/AzureDevOps/StageTemplates/RunTests.yml

```diff
@@ -0,0 +1,136 @@
+parameters:
+  AzureSubscription: ''
+  CodeCoveragePercentage: 80
+
+stages:
+- stage: Test
+  jobs:
+    - job: RunAcceptanceTests
```

Comment:
There's an argument for refactoring some of this into a job template but I think that should be left until after this PR has been merged back to master given how much has changed already

Reply:
I'll add a task to this case to do that so it doesn't get lost

Reply:
https://skillsfundingagency.atlassian.net/browse/NCSD-2615